### PR TITLE
[Refactor] Refactor access broker error log

### LINKED ISSUE #5123: BACKUP出错
### 备份操作
BACKUP SNAPSHOT test.update_detail
TO hdfs_repo1
ON(update_detail)
PROPERTIES ("type" = "full","timeout" = "3600");

### Expected behavior (Required)
我备份的是一个更新模型的空表
c0bb34ac123113474df1734fcaa4167
出现报错
image
帮忙看下是什么问题

### StarRocks version (Required)
 - starrocks2.1

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #5123

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: be/src/fs/fs_broker.cpp [hunk 1]
```diff
@@ -332,7 +332,7 @@ StatusOr<std::unique_ptr<WritableFile>> BrokerFileSystem::new_writable_file(cons
         return Status::NotSupported("Open with MUST_EXIST not supported by broker");
     } else if (opts.mode == CREATE_OR_OPEN) {
         if (auto st = _path_exists(path); st.ok()) {
-            return Status::NotSupported("Cannot write an already exists file through broker");
+            return Status::NotSupported("Cannot write an already exists file through broker, path="_format(path));
```
Comment (imay) [inline be/src/fs/fs_broker.cpp:335]: Status::AlreadyExist

Comment (meegoo) [reply be/src/fs/fs_broker.cpp]: opts.mode is CREATE_OR_OPEN, return error since broker doesn't support open an already exists file.

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by xiaoyong-z** [comment]
LGTM

**Review by chaoyli — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**